.github/workflows/deploy-staging.yml

```
@@ -0,0 +1,93 @@
+on:
+  push:
+    branches:
+      - staging
+  pull_request:
+    branches:
+      - staging
+
+jobs:
+  test:
+    name: Run Tests
+    uses: ./.github/workflows/test.yml
```

actionlint (1.7.7)

could not read reusable workflow file for "./.github/workflows/test.yml": open /home/jailuser/git/.github/workflows/test.yml: no such file or directory

(expression)